modules/combined/tests/contact_verification/patch_tests/plane_3/plane3_template1.i

```diff
@@ -269,8 +269,8 @@
   #Preconditioned JFNK (default)
   solve_type = 'PJFNK'
 
-  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
-  petsc_options_value = 'lu     superlu_dist'
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package -mat_superlu_dist_fact'
+  petsc_options_value = 'lu            superlu_dist               SamePattern'
```

Comment:
The option has to be applied to some many examples? I remember there are three examples only, you guys said last time.  We may add this option in moose directly instead of every single example.